README.md

```diff
@@ -40,10 +40,20 @@ CMR-GraphQL uses a few environment variables for configuring runtime options:
 |MMT_ROOT_URL||URL to ping when retrieving metadata from MMT e.g. https://mmt.earthdata.nasa.gov|
 |DRAFT_MMT_ROOT_URL||URL to ping when retrieving draft metadata from Draft MMT e.g. https://draftmmt.earthdata.nasa.gov|
 |LAMBDA_TIMEOUT|30|Number of seconds to set the Lambda timeout to.|
-|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID|For facilitating EDL connection -- obtain these from a dev|
+|EDL_KEY_ID, EDL_JWK, EDL_CLIENT_ID, EDL_PASSWORD| .bash or .zshrc file|For facilitating EDL connection -- obtain these from a dev and upload. After making changes, remember to run $ source ~/.zshrc|
```

Comment:
`.bash or .zshrc file` is not a default value that is used, so it doesn't make sense in that column. And again, there are more than one valid ways to save environment variables, adding that you have to run `$ source ~/.zshrc` is not valid for all users, and we should leave it up to the dev to set variables however they want to